### PR TITLE
Disable tests of known broken modules

### PR DESCRIPTION
**Pull request by mmazur**
Until we have separate jobs for devel and stable ansible.

### CODE REVIEW

File: automation/test.sh [hunk 1]
```diff
@@ -27,6 +27,9 @@ pip3 install --user ansible
 ansible-playbook --version
 ansible-playbook -vvv tests/playbooks/all.yml
 
+# TODO: REMOVE THIS LATER
```
Comment (pkliczewski) [inline automation/test.sh]: This seems to prevent us to get any error. It would be better not to test broken code for devel branch only.

Comment (machacekondra) [reply automation/test.sh]: Yeah, can we just use github API to fetch issue and skip the test if the issue is open and don't skip if it's closed, that way we now it was fixed properly and we will be testing everything working always

**Comment by mmazur** [comment]
Ok, this patch disables the borked tests on dev ansible. I need to do other stuff first, before I can add the issue–detection logic, so please merge this for now and I'll send the rest in another PR.

**Comment by pkliczewski** [comment]
@machacekondra what do you think?

**Pull request merged**